dynamics-nav/Converting-a-Database.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "Converting a Database - Technical Upgrade"
+description: "This article describes how to convert a Dynamics NAV database to Microsoft Dynamics NAV 2018."
```

Comment:
Too short per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.